etc/dolphin/dolphin.conf

```diff
@@ -1,5 +1,5 @@
 [DEFAULT]
-api_paste_config = C:\pythonpath\dolphin\etc\dolphin\api-paste.ini
+api_paste_config = /opt/naju/katurii/dolphin/etc/dolphin/api-paste.ini
```

Comment:
why this custom and hardcoded path?

Reply:
this we will improve.. as of now for dev evry one has their spec..

Reply:
ok